---
layout: post
title:  "Training Course on RNA-Seq data analysis"
date_start:   2017-07-18
date_end:   2017-07-20
venue: Milano, Italy
description: The course will provide students with theoretical and practical knowledge on how to perform bioinformatic analyses of RNA-Seq data, mainly aimed at the identification of genes differentially expressed across different conditions. 

---

<table border="0" width="600">
  <tr>
    <td><img src="../../../img/logo_elixir_italy.jpg" height="100"></td>
    <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>
    <td><img src="../../../img/Logo_unimilano.jpg" height="100"></td>
    <td><img src="../../../img/Logo_dbs_milano.jpg" height="100"></td>
  </tr>
</table>
<br>

**IMPORTANT DATES** for this Training course:

- Deadline for applications: 30 June 2017
- Course date: 18-20 July 2017

**VENUE:**
Computer Room (aula Calcolo) Department of Biosciences, University of Milan (Via Celoria 26, Milan, Italy) 

**FEE:** 
The course does not include a fee but the participants are expected to pay their own travel, meals and hotel costs (if any).

A maximum of **30** candidates will be accepted in the course. Selection will start on July 1st  based on the candidates’ profile. Priority will be given to PhD students of the University of Milan and to candidates coming from abroad. 
<br>

### Instructors
- [**Giulio Pavesi**](http://www.beaconlab.it) - Dept. of Biosciences, University of Milano
- [**Fabrizio Ferré**](https://www.unibo.it/sitoweb/fabrizio.ferre) - Dept. of Pharmacy and Biotechnology, University of Bologna
- [**Federico Zambelli**](http://www.beaconlab.it) - Dept. of Biosciences, University of Milano
- [**Matteo Chiara**](http://www.beaconlab.it) - Dept. of Biosciences, University of Milano
<br>

### Organisers
- Giulio Pavesi (Dept. of Biosciences, University of Milano)
- Federico Zambelli (Dept. of Biosciences, University of Milano and ELIXIR-IIB Technical Coordinator)
- Matteo Chiara (Dept. of Biosciences, University of Milano)
- Allegra Via (ELIXIR-IIB Training Coordinator, CNR, Italy)
<br>

### Course Description
The course will provide students with theoretical and practical knowledge on how to perform bioinformatic analyses of RNA-Seq data, mainly aimed at the identification of genes differentially expressed across different conditions. 
<br>



### Target audience
PhD Students or young researchers in molecular biology and/or genetics with little or no background in bioinformatics.
<br>

### Learning outcomes
After this course participants should be able to:
<ul><li>understand the theoretical principles of the analysis of RNA-Seq data</li>
<li>plan and execute the main steps of a bioinformatic RNA-Seq data analysis</li>
<li>use statistical approaches for the identification of differentially expressed genes</li>
<li>understand the results obtained</li>
</ul>


### Course prerequisites
Basic background knowledge on genetics, molecular biology and transcriptomics.

<br>
### [Application Form](https://goo.gl/forms/L2zHQVDTap055ZMz1) 
<br>

### Programme

<table border="1">
<tr>
   <td colspan="4"><h3>Tuesday - 18 July 2017</h3></td>
</tr>
<tr>
   <td height="50">14:00 - 14:30</td>
   <td colspan="3" height="50">Opening and Welcome. Participants’ self-presentation</td>
</tr>
<tr>
   <td height="50">14:30 - 15:00</td>
   <td height="50">Lecture</td>
   <td height="50">Giulio Pavesi</td>
   <td height="50">Introduction to NGS and RNA-Seq</td>
</tr>
<tr>
   <td height="50">15:00 - 16:00</td>
   <td height="50">Lecture</td>
   <td height="50">Matteo Chiara</td>
   <td height="50">Theoretical principles in the analysis of RNA-Seq data.</td>
</tr>
<tr>
   <td height="50">16:00 - 16:30</td>
   <td colspan="3"  height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">16:30 - 17:30</td>
   <td height="50">Lecture</td>
   <td height="50">Federico Zambelli</td>
   <td height="50">Introduction to the Galaxy Platform</td>
</tr>
<tr>
   <td colspan="4"><h3>Wednesday - 19 July 2017</h3></td>
</tr>
<tr>
  <td height="50">09:30 - 11:00</td>
  <td height="50">Practical</td>
  <td height="50">Fabrizio Ferré</td>
  <td height="50">Quality control, read mapping</td>
</tr>
<tr>
   <td height="50">11:00 - 11:30</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:30 - 13:00</td>
   <td height="50">Practical</td>
   <td height="50">Fabrizio Ferré</td>
   <td height="50">Read mapping and estimation of transcript levels</td>
</tr>
<tr>
   <td height="50">13:00 - 14:00</td>
   <td colspan="3" height="50"> Lunch break </td>
</tr>
<tr>
   <td height="50">14:00 - 16:00</td>
   <td height="50">Practical </td>
   <td height="50">Fabrizio Ferré</td>
   <td height="50">Identification of differentially expressed genes</td>
</tr>
<tr>
   <td height="50">16:00 - 16:30</td>
   <td colspan="3" height="50"> Coffe break </td>
</tr>
<tr>
   <td height="50">16:30 - 18:00</td>
   <td height="50">Practical </td>
   <td height="50">Fabrizio Ferré</td>
   <td height="50">Functional characterization of differentially expressed genes</td>
</tr>
<tr>
<td colspan="4"><h3>Thursday - 20 July 2017</h3></td>
</tr>
<tr>
<td height="50">09:30 - 11:00</td>
<td height="50">Lecture</td>
<td height="50">Giulio Pavesi</td>
<td height="50">Guidelines and strategies for post-processing of the results</td>
</tr>
<tr>
<td height="50">11:00 - 12:00</td>
<td colspan="3" height="50">Wrap-up and feedback</td>
</tr>
</table>

<br>

<h3>ELIXIR-IIB member institutions</h3>
<ul>
   <li> CNR (ELIXIR-IIB coordinator)</li>
   <li> CRS4</li>
   <li> CINECA</li>
   <li> Fondazione Edmund Mach, Trento</li>
   <li> INFN</li>
   <li> GARR</li>
   <li> Sapienza Università di Roma</li>
   <li> Università di Bari</li>
   <li> Università di Bologna</li>
   <li> Università di Firenze</li>
   <li> Università di Milano</li>
   <li> Università di Milano Bicocca</li>
   <li> Università di Padova</li>
   <li> Università di Parma</li>
   <li> Università di Roma "Tor Vergata"</li>
   <li> Università di Salerno</li>
   <li> Università della Tuscia </li>
</ul>


