---
layout: post
title:  "Training course on Python for Life Scientists <font color='black'>[closed]</font>"
date_start:   2017-09-11
date_end:   2017-09-15
venue: Roma, Italy
description: Python is an object-oriented programming language that is ideal for biological data analysis. The course will start from zero knowledge, and will introduce the participants to all the basic concepts of Python such as calculating, organising data, reading and writing files, program logic and writing larger programs. All the examples and practical sessions will focus on solving biological problems. The course will be highly interactive and the students will continuously put theory into practice while learning.

---

<table border="0" width="600">
  <tr>
    <td><a href="http://elixir-italy.org"><img src="../../../img/logo_elixir_italy.jpg" height="100"></a></td>
    <td></td>
    <td><a href="http://www.uniroma1.it/"><img src="../../../img/Logo_sapienza.jpg" height="80" width="250"></a></td>
  </tr>
</table>
<br>

**IMPORTANT DATES** for this Training course:

- Deadline for applications: 20 Aug 2017
- Course date: 11-15 Sep 2017
<br>

**VENUE:**
Sapienza University of Rome, Department of Biochemical Science A. Rossi Fanelli, p.le Aldo Moro 5, 00185 Roma, Italy.
<br>

**FEE:** 
The course does not include a fee, but the participants are expected to pay their own travel, meals and hotel costs (if any).

A maximum of **20** candidates will be selected based on their research profile. Notifications of acceptance will be sent shortly after the application until we reach 20 participants. Priority will be given to candidates from ELIXIR-IIB member institutions (see the list at the bottom) and ELIXIR nodes. 
<br>

### Instructors
- [**Allegra Via**](../../../instructors/allegra_via.html) - ELIXIR-IIB Training Coordinator, IBPM-CNR, Italy
- [**Alberto Calderone**](../../../instructors/alberto_calderone.html) - University of Rome Tor Vergata, Italy
- [**Giacomo Janson**](../../../instructors/giacomo_janson.html) - Dept. of Biochemical Sciences, Sapienza University of Rome, Italy

### Helpers
- Elisa Micarelli, University of Rome "Tor Vergata", Italy

### Organisers 
- Veronica Morea (IBPM-CNR, Rome, Italy)
- Allegra Via (ELIXIR-IIB Training Coordinator, IBPM-CNR, Italy)
- Loredana Le Pera (ELIXIR-IIB Training Team, CLNS@Sapienza, Italy)
<br>

### Course Description
Python is an object-oriented programming language that is ideal for biological data analysis. The course will start from zero knowledge, and will introduce the participants to all the basic concepts of Python such as calculating, organizing data, reading and writing files, program logic and writing larger programs. All the examples and practical sessions will focus on solving biological problems. 
In particular the sessions will cover:
- working with DNA and protein sequences
- data retrieval from files and their manipulation
- running applications, such as BLAST, locally and from a script
- finding motifs in sequence
- parsing common file formats (Uniprot, GenBank, PDB)
- ways to find and correct program errors

The course will be highly interactive and the students will continuously put theory into practice while learning. Basic Unix/Linux skills will be provided at the beginning of the course.

### Target audience
End-users of bioinformatics databases and tools that aim at developing hands-on capabilities for biological data analysis, ie writing their own or adapting somebody else's Python scripts in an autonomous way.
<br>

### Learning outcomes
By the end of the course, the participants will have a good understanding of the Python basics and will have acquired the skills to manage any type of bioinformatics database record, to manipulate data files and tables, and to run applications from scripts.

### Course prerequisites
Basic familiarity with bioinformatics data resources such as Uniprot/Swiss-Prot, Blast, ENSEMBL, PDB, etc. The course is directed to biologists with little or no programming experience and aims at making them capable to use Python to autonomously manage and analyse biological data.
<br>
<br>

### [Application Form](https://goo.gl/forms/TVfcESvpljDQm7Tl2) 
<br>
<br>


### Programme

<table border="1">
<tr>
   <td colspan="4"><h3>Monday - 11 Sep 2017 - Sharpening Your Knives</h3></td>
</tr>
<tr>
   <td height="50">09:00 - 09:45</td>
   <td colspan="3" height="50">Welcome, intro & expectations.</td>
</tr>
<tr>
   <td height="50">09:45 - 11:15</td>
   <td height="50">Long chat</td>
   <td height="50">Intro to the Linux command line</td>
</tr>
<tr>
   <td height="50">11:15 - 11:45</td>
   <td colspan="3"  height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:45 - 13:15</td>
   <td height="50">Practical</td>
   <td height="50">Linux tutorial</td>
</tr>
<tr>
   <td height="50">13:15 - 14:30</td>
   <td colspan="3" height="50">Lunch break </td>
</tr>
<tr>
  <td height="50">14:30 - 16:00</td>
  <td height="50">Lecture & Practical</td>
  <td height="50">Python Shell</td>
</tr>
<tr>
   <td height="50">16:00 - 16:30</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
  <td height="50">16:30 - 18:00</td>
  <td height="50">Lecture & Practical</td>
  <td height="50">Python programs</td>
</tr>
<tr>
   <td colspan="4"><h3>Tuesday - 12 Sep 2017 - Manipulating Things</h3></td>
</tr>
<tr>
   <td height="50">09:00 - 10:30</td>
   <td height="50">Lecture & Practical</td>
   <td colspan="3" height="50"> Repeating things </td>
</tr>
<tr>
   <td height="50">10:30 - 11:00</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:00 - 12:45</td>
   <td height="50">Lecture & Practical</td>
   <td height="50"> Reading and writing files</td>
</tr>
<tr>
   <td height="50">12:45 - 14:15</td>
   <td colspan="3" height="50"> Lunch break </td>
</tr>
<tr>
   <td height="50">14:15 - 16:00</td>
   <td height="50">Lecture & Practical</td>
   <td height="50">Parsing </td>
</tr>
<tr>
   <td height="50">16:00 - 16:30</td>
   <td colspan="3" height="50"> Coffe break </td>
</tr>
<tr>
   <td height="50">16:30 - 18:00</td>
   <td height="50">Lecture & Practical</td>
   <td height="50">Parsing </td>
</tr>
<tr>
<td colspan="4"><h3>Wednesday - 13 Sep 2017 - Creating and assessing Things</h3></td>
</tr>
<tr>
   <td height="50">09:00 - 10:30</td>
   <td height="50">Lecture & Practical</td>
   <td height="50">Functions </td>
</tr>
<tr>
   <td height="50">10:30 - 11:00</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:00 - 12:45</td>
   <td height="50">Lecture & Practical</td>
   <td height="50">Debugging and Error Handling</td>
</tr>
<tr>
   <td height="50">12:45 - 14:15</td>
   <td colspan="3" height="50"> Lunch break </td>
</tr>
<tr>
<td height="50">14:15 - 16:00</td>
<td height="50">Lecture & Practical</td>
<td height="50">Data columns (formatting and sorting)</td>
</tr>
<tr>
   <td height="50">16:00 - 16:30</td>
   <td colspan="3" height="50"> Coffee break </td>
</tr>
<tr>
<td height="50">16:30 - 18:00</td>
<td height="50">Lecture & Practical</td>
<td height="50">Tabular Data</td>
</tr>
<tr>
<td colspan="4"><h3>Thursday - 14 Sep 2017 - Connecting Things</h3></td>
</tr>
<tr>
   <td height="50">09:00 - 10:30</td>
   <td height="50">Lecture & Practical</td>
   <td height="50">Searching </td>
</tr>
<tr>
   <td height="50">10:30 - 11:00</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:00 - 12:45</td>
   <td height="50">Lecture & Practical</td>
   <td height="50">Pipelines </td>
</tr>
<tr>
   <td height="50">12:45 - 14:15</td>
   <td colspan="3" height="50"> Lunch break </td>
</tr>
<tr>
<td height="50">14:15 - 16:00</td>
<td height="50">Practical</td>
<td height="50">Python Libraries</td>
</tr>
<tr>
   <td height="50">16:00 - 16:30</td>
   <td colspan="3" height="50"> Coffee break </td>
</tr>
<tr>
<td height="50">16:30 - 18:00</td>
<td height="50">Practical</td>
<td height="50">Python Libraries</td>
</tr>
<tr>
<td colspan="4"><h3>Friday - 15 Sep 2017 - Developing a coding project in groups</h3></td>
</tr>
<tr>
   <td height="50">09:00 - 10:30</td>
   <td height="50">Practical</td>
   <td height="50">Intro to the Hackathon</td>
</tr>
<tr>
   <td height="50">10:30 - 11:00</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:00 - 12:45</td>
   <td height="50">Practical</td>
   <td height="50">Hackathon</td>
</tr>
<tr>
   <td height="50">12:45 - 14:00</td>
   <td colspan="3" height="50"> Lunch break </td>
</tr>
<tr>
<td height="50">14:00 - 15:30</td>
<td height="50">Practical</td>
<td height="50">Hackathon</td>
</tr>
<tr>
   <td height="50">15:30 - 16:00</td>
   <td colspan="3" height="50"> Coffee break </td>
</tr>
<tr>
   <td height="50">16:00 - 17:30</td>
   <td colspan="3" height="50"> Hackathon presentations. Expectations, outcomes, feedback. Wrap up. </td>
</tr>
</table>

<br>

<h3>ELIXIR-IIB member institutions</h3>
<ul>
   <li> CNR (ELIXIR-IIB coordinator)</li>
   <li> CRS4</li>
   <li> CINECA</li>
   <li> Fondazione Edmund Mach, Trento</li>
   <li> INFN</li>
   <li> GARR</li>
   <li> Sapienza Università di Roma</li>
   <li> Università di Bari</li>
   <li> Università di Bologna</li>
   <li> Università di Firenze</li>
   <li> Università di Milano</li>
   <li> Università di Milano Bicocca</li>
   <li> Università di Padova</li>
   <li> Università di Parma</li>
   <li> Università di Roma "Tor Vergata"</li>
   <li> Università di Salerno</li>
   <li> Università della Tuscia </li>
</ul>


