---
layout: post
title:  Workshop on "Computational approaches to the study of protein interactions and rational drug design" <font color='black'>[closed]</font>
date_start:   2017-04-10
date_end:   2017-04-13
venue: Padova, Italy
description: This workshop is aimed at students in master degree courses, PhD students and post-doctoral researchers who are working or planning to work in the field of protein interactions from a molecular and structural point of view. In this workshop, participants will familiarise with docking approaches to the study of protein-protein interactions (PPIs) and protein-small molecule interactions. Irreversible binders design using molecular docking and computational methods to rational drug design will be also described, as well as the functional role of the intrinsically disordered proteins.

---

<table border="0" width="600">
  <tr>
    <td><img src="../../../img/logo_elixir_italy.jpg" height="100"></td>
    <td><img src="../../../img/logo_uni_padova.png" height="100"></td>
    <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>
    <td><img src="../../../img/logo_rsg_italy.jpg" height="100"></td>
  </tr>
</table>

<br>

**IMPORTANT DATES** for this Training course:

- Deadline for applications: 31st March 2017 
- Course date: 10-13 April 2017

**VENUE**:
Campus di Biologia e Biomedicina "Fiore di Botta", University of Padova, Via del Pescarotto, 8, 35131 Padova, IT

**FEE**: 
The fee will cover attendance to lectures and practicals, coffee breaks, lunches, and the welcome spritz on Monday, 10th April. The participants are expected to pay their own travel and accommodation costs.
 
 - Candidates from a bioinformatics lab from one of the ELIXIR-ITA member institutions (see the list at the bottom): **130 euros**. 
 - All the other candidates: **180 euros**

A maximum of 30 candidates will be accepted in the course. Selection will start on March the 10th and those with an adequate profile will be accepted immediately, especially if they come from other countries (to allow them to find reasonably cheap flight tickets). 

### Speakers
- [**Alexandre Bonvin**](../../../instructors/alexandre_bonvin.html) - Faculty of Science, Utrecht University, the Netherlands
- [**Norman Davey**](../../../instructors/norman_davey.html) (University College Dublin, Dublin, Ireland)
- [**Stefano Forli**](../../../instructors/stefano_forli.html) (Dept. Molecular Biology, The Scripps Research Institute, La Jolla, USA)
- [**Stefano Moro**](http://mms.dsfarm.unipd.it/smoro.html) (Dept. of Pharmaceutical and Pharmacological Sciences, University of Padova, Padova, Italy)

### Organisers
- Marco Carraro (University of Padova, Italy)
- Lisanna Paladin (University of Padova, Italy)
- Allegra Via (ELIXIR-IIB Training Coordinator, CNR, Italy)

### Course Description
In this workshop, participants will familiarise with docking approaches to the study of protein-protein interactions (PPIs) and protein-small molecule interactions. Irreversible binders design using molecular docking and computational methods to rational drug design will be also described, as well as the functional role of the intrinsically disordered proteins.
The Plenary lectures of N. Davey (10.02.2017), A. Bonvin, S. Forli and S. Moro (11.02.2017) will be open and free to everyone but require application. <br>
Practical sessions are limited to 30 participants. 
On the afternoon of day 3, participants can bring their own data (e.g. proteins they want to study interactions for). They will be split in groups based on the type of system they want to study (i.e protein-protein, protein-small ligand interactions or virtual screening) and will analyse it assisted by A. Bonvin and S. Forli.

### Target audience

Students and researchers in the life sciences / structural biology / computational biology field, interested in structure-based drug design, and in the functional subdivision of proteins into modules and how these modules co-operate to govern the spatiotemporal functionality of the protein. Students and researchers interested in general to structure-based ligand design and, or specifically, to docking-driven hit/lead identification.

### Learning objectives

Participants will:<br>

- gain an understanding of 
	- the modular architecture of a protein; 
	- the functional role of post-translation modification, short linear motifs and globular domains; 
	- and the complex regulatory functionality that is encoded in proteins at the sequence level.

They will also learn:

- how to transform generic information about binding sites into restraints to drive docking in HADDOCK
- the basics of information-driven docking using the HADDOCK web server
- how to perform protein-ligand docking using HADDOCK (and screening for putative binding sites)
- to critically look at docking results
- the basic principles of small molecules docking
- about the potential of the method, its scope and application, and be aware of its limitation
- how to perform single ligand dockings, as well as design and perform larger virtual screening experiments
- how to compare the performance of different docking protocols
- how to select the best performant docking protocol
- when is better to be aware of docking results

### Course prerequisites
- Basic knowledge of protein structure
- Basic Linux command knowledge

### [Application Form](https://goo.gl/forms/jWSsU0Uv9FS6Ykqs1)

### Programme

<table border="1">
<tr>
   <td colspan="4"><h3>Monday 10th Apr 2017</h3></td>
</tr>
<tr>
   <td>13.30-14.30</td>
   <td>Plenary lecture</td>
   <td>Norman Davey</td>
   <td><a href="https://elixir-iib-training.github.io/website/docs/norman-davey-abstract-padova-2017.pdf">Understanding the functional role of the intrinsically disordered proteome</a></td>
</tr>
<tr>
   <td>14:30 -15:00</td>
   <td>Course opening</td>
   <td colspan="2">Participants’ self-presentations</td>
</tr>
<tr>
   <td>15:00-15:45</td>
   <td>Practical</td>
   <td>Norman Davey</td>
   <td>Tools and resources for the analysis of the structural architecture of a protein and the functional role of protein globular domains and disordered regions.</td>
</tr>
<tr>
   <td>15:45-16:15</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>16.15-18:30</td>
   <td>Practical</td>
   <td>Norman Davey</td>
   <td>Tools and resources for the analysis of the regulatory mechanisms controlling the function of a protein and the regulatory program encoded by the full complement of functional modules.
 </td>
</tr>
<tr>
   <td>19:30-21:00</td>
   <td colspan="3">Welcome cocktail with finger food at Verba Volant</td>
</tr>
<tr>
   <td colspan="4"><h3>Tuesday 11th Apr 2017</h3></td>
</tr>
<tr>
  <td>9:30-10:30</td>
  <td>Plenary lecture</td>
  <td>Alexandre Bonvin</td>
  <td><a href="https://elixir-iib-training.github.io/website/docs/alexandre-bonvin-abstract-padova-2017.pdf">High-resolution, integrative modelling of biomolecular complexes from fuzzy data</a></td>
</tr>
<tr>
   <td>10:30-11:00</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>11:00-12:00</td>
   <td>Plenary lecture</td>
   <td>Stefano Forli</td>
   <td><a href="https://elixir-iib-training.github.io/website/docs/stefano-forli-abstract-padova-2017.pdf"> Irreversible binders design using molecular docking: the Reactive Docking method </a></td>
</tr>
<tr>
   <td>12:00-13:00</td>
   <td>Plenary lecture</td>
   <td>Stefano Moro</td>
   <td><a href="https:/elixir-iib-training.github.io/website/docs/stefano-moro-abstract-padova-2017.pdf">To dock, or not to dock… how to deal with the quality assessment strategy of  protein-ligand docking protocols comparing their performances</a></td>
</tr>
<tr>
   <td>13:00-14:30</td>
   <td colspan="3"> Lunch </td>
</tr>
<tr>
   <td>14:30-16:15</td>
   <td> Practical </td>
   <td>Alexandre Bonvin</td>
   <td><a href="http://www.bonvinlab.org/education/HADDOCK-protein-protein-basic/">Basic protein-protein docking tutorial</a></td>
</tr>
<tr>
<td>16:15-16:45</td>
<td colspan="3">Coffee Break</td>
<tr>
</tr>
<td>16:45-18:30</td>
<td>Practical</td>
<td>Alexandre Bonvin</td>
<td><a href="http://www.bonvinlab.org/education/HADDOCK-binding-sites/">Protein-ligand docking tutorial</a></td>
</tr>
<tr>
<td colspan="4"><h3>Wednesday 12th Apr 2017</h3></td>
</tr>
<tr>
<td>9:15 - 11:00</td>
<td>Practical</td>
<td>Stefano Forli</td>
<td>Introduction to ligand-protein docking with AutoDock</td>
</tr>
<tr>
<td>11:00-11:30</td>
<td colspan="3">Coffee Break</td>
</tr>
<tr>
<td>11:30-13:00</td>
<td>Practical</td>
<td>Stefano Forli</td>
<td>Virtual Screening with AutoDock Raccoon2</td>
</tr>
<tr>
<td>13:00-14:30</td>
<td colspan="3">Lunch</td>
</tr>
<tr>
<td>14:30-16:00</td>
<td>BYOD session</td>
<td colspan="2">A. Bonvin and S. Forli</td>
</tr>
<tr>
<td>16:00-16:30</td>
<td colspan="3">Coffee Break</td>
</tr>
<tr>
<td>16:30-18:30</td>
<td>BYOD session</td>
<td colspan="2">A. Bonvin and S. Forli</td>
</tr>
<tr>
<td colspan="4"><h3>Thursday 13th Apr 2017</h3></td>
</tr>
<tr>
<td>9.15-10:00</td>
<td>Practical</td>
<td>Stefano Moro</td>
<td>10 more tips to deal with molecular docking: the practical session will be devoted to introduce, to a non-expert audience, the basic principle of quality assessment of molecular docking performances </td>
</tr>
<tr>
<td>10:00-10:30</td>
<td colspan="3">Coffee Break</td>
</tr>
<tr>
<td>10:30-12:30</td>
<td>Practical</td>
<td>Stefano Moro</td>
<td>10 more tips to deal with molecular docking</td>
</tr>
<tr>
<td>12:30-13:00</td>
<td colspan="3">Wrap-up and feedback</td>
</tr>
</table>

<p>

<h3>ELIXIR-ITA member institutions</h3>

<ul>
   <li> CNR (ELIXIR-ITA coordinator)
   <li> CRS4
   <li> CINECA
   <li> Fondazione Edmund Mach, Trento
   <li> INFN
   <li> GARR
   <li> Sapienza Università di Roma
   <li> Università di Bari
   <li> Università di Bologna
   <li> Università di Firenze
   <li> Università di Milano
   <li> Università di Milano Bicocca
   <li> Università di Padova
   <li> Università di Parma
   <li> Università di Roma "Tor Vergata"
   <li> Università di Salerno
   <li> Università della Tuscia 
</ul>


