---
layout: post
title:  "Training Course on Best practices for RNA-Seq data analysis"
date_start:   2017-09-27
date_end:   2017-09-29
venue: Salerno, Italy
description: The aim of this course is to get a deeper understanding of RNA-Seq experiments, providing a theoretical introduction to the data processing steps, together with practical sessions illustrating the use of the most popular data analysis tools. Some strategies are illustrated for detecting genes, which are differentially expressed across different biological conditions, alternative splicing products, novel isoforms and gene fusion events. Some methods for downstream analysis are described to give insight into how biological knowledge can be generated from RNA-Seq experiments.

---

<table border="0" width="600">
  <tr>
    <td><a href="http://elixir-italy.org"><img src="../../../img/logo_elixir_italy.jpg" height="100"></a></td>
    <td></td>
    <td><a href="http://web.unisa.it/ateneo"><img src="../../../img/Logo_unisalerno.png" height="100"></a></td>
  </tr>
</table>
<br>

**IMPORTANT DATES** for this Training course:

- Deadline for applications: 11 Sep 2017
- Course date: 27-29 Sep 2017
<br>

**VENUE:**
Campus di Fisciano, University of Salerno, Via Giovanni Paolo II n. 132, 
84084 Fisciano (SA), Italy. 

<!--  To get in:
 - By train: Trenitalia (national railways) stops at 
 - By car: The venue is 
 - By fly:

SUGGESTED ACCOMMODATIONS:
A limited number of rooms will be availables at the venue (costs 20 EUR/night). Please contact -->
<br>

**FEE:** 
The fee will cover attendance to lectures and practicals, coffee breaks, lunches and dinners. The participants are expected to pay their own travel and accommodation costs:
- candidates from a bioinformatics lab from one of the ELIXIR-IIB member institutions (see the list at the bottom): **110 euros**
- other candidates: **160 euros**

A maximum of **25** candidates will be accepted. Selection will start on July 15th 2017 and those with an adequate profile will be accepted immediately. 
<br>

### Instructors
- [**Fabrizio Ferré**](https://www.unibo.it/sitoweb/fabrizio.ferre) - Dept. of Pharmacy and Biotechnology, University of Bologna, Italy
- **Loredana Le Pera** - Center for Life Nano Science@Sapienza, IIT, Roma, Italy
- **Giorgio Giurato** - Genomix4Life, Italy
- **Allegra Via** - (ELIXIR-IIB Training Coordinator, CNR-IBPM, Italy) 
<br>

### Organisers 
- Anna Marabotti (University of Salerno and ELIXIR-IIB Training Team, Italy)
- Roberto Tagliaferri (University of Salerno, Italy)
- Alessandro Weisz (University of Salerno, Italy)
- Loredana Le Pera (ELIXIR-IIB Training Team and IIT, Italy) 
- Allegra Via (ELIXIR-IIB Training Coordinator, CNR-IBPM, Italy)
<br>

### Course Description
Next-generation sequencing (NGS) of RNA libraries (RNA-Seq) has become increasingly common and it largely replaced microarray technology for gene expression profiling. The aim of this course is to get a deeper understanding of RNA-Seq experiments, providing a theoretical introduction to the data processing steps, together with practical sessions illustrating the use of the most popular data analysis tools. The classroom size is limited to 25 participants to optimize the learning and the interaction with the instructors. Starting from the raw sequenced data coming from different phenotypical samples (e.g disease vs healthy control samples), genes which are differentially expressed between the two conditions are determine. Some strategies are illustrated for detecting alternative splicing products, predicting novel isoforms and gene fusion events. Some methods for downstream analysis are described to give insight into how biological knowledge can be generated from RNA-Seq experiments. A lecture on single-cell RNA-Seq will provide an idea of how transcriptome data from individual cells is now emerging as a powerful tool, allowing the study of cell-to-cell gene expression heterogeneity in the same tissue. 

### Course objectives
Students will gain an understanding of: 
- experimental design
- quality control 
- theoretical principles of RNA-Seq data analysis process
- selection of differentially regulated genes
- selection of alternative splicing events
- identification of gene fusion events
- multiple testing correction
- biological interpretation of RNA-seq data

### Target audience
This course is aimed at PhD students and young researchers in the life science and computational biology field who are planning to use RNA-seq data and are looking for the best practices to analyze these types of data. Basic knowledge of UNIX/Linux operative system and the BASH command line is desirable; however, on the first day of the course, a practical tutorial will provide an introduction about shell usage and common commands. To apply for the course please fill in the application form, which includes a questionnaire; participants will be selected according to their profile and to their answers to the questionnaire. 
<br>

### Learning outcomes
After this course participants should be able to:
<ul><li>understand the importance of RNA-Seq experimental design</li>
<li>assess the quality of data</li>
<li>preprocess data</li>
<li>align data to a reference genome/transcriptome</li>
<li>perform a complete analysis of RNA-Seq data (Tuxedo Suite)</li>
<li>estimate known gene and transcript expression</li>
<li>discover alternative splicing events and novel isoforms</li>
<li>discover fusion gene events</li>
<li>perform differential expression analysis</li>
<li>perform functional enrichment analysis</li>
<li>summarize and interpret the RNA-seq analysis results</li>
</ul>

### Course prerequisites
A basic understanding of molecular biology and transcriptomics is assumed. A basic knowledge of Linux (shell usage, common commands) and familiarity with elementary statistics are beneficial, but not essential. 
<br>
<br>

### [Application Form](https://goo.gl/forms/) 
<font color="red"><b>(it will be open from July 7th)</b></font>
<br>
<br>


### Programme

<table border="1">
<tr>
   <td colspan="4"><h3>Wednesday - 27 Sep 2017</h3></td>
</tr>
<tr>
   <td height="50">09:15 - 10:00</td>
   <td colspan="3" height="50">Welcome and Opening. Participants' self-presentation</td>
</tr>
<tr>
   <td height="50">10:00 - 11:00</td>
   <td height="50">Lecture</td>
   <td height="50"></td>
   <td height="50">An introduction to NGS and RNA-Seq </td>
</tr>
<tr>
   <td height="50">11:00 - 11:30</td>
   <td colspan="3"  height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:30 - 13:00</td>
   <td height="50">Lecture</td>
   <td height="50"></td>
   <td height="50">Intro to the Linux command line</td>
</tr>
<tr>
   <td height="50">13:00 - 14:00</td>
   <td colspan="3" height="50">Lunch break </td>
</tr>
<tr>
  <td height="50">14:00 - 14:45</td>
  <td height="50">Lecture</td>
  <td height="50"></td>
  <td height="50">Overview of the RNA-Seq data analysis process: from raw data (reads) to the differentially expressed gene list</td>
</tr>
<tr>
  <td height="50">14:45 - 15:30</td>
  <td height="50">Lecture</td>
  <td height="50"></td>
  <td height="50">Preprocessing of RNA-Seq raw data: quality control, trimming, filtering</td>
</tr>
<tr>
   <td height="50">15:30 - 16:00</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
  <td height="50">16:00 - 18:30</td>
  <td height="50">Practical</td>
  <td height="50"></td>
  <td height="50">Linux tutorial - Raw data preprocessing </td>
</tr>
<tr>
   <td colspan="4"><h3>Thursday - 28 Sep 2017</h3></td>
</tr>
<tr>
   <td height="50">09:15 - 09:30</td>
   <td colspan="3" height="50"> Wrap up </td>
</tr>
<tr>
  <td height="50">09:30 - 10:00</td>
  <td height="50">Lecture</td>
  <td height="50"></td>
  <td height="50">Alignment of RNA-Seq data</td>
</tr>
<tr>
   <td height="50">10:00 - 10:30</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">10:30 - 13:00</td>
   <td height="50">Practical</td>
   <td height="50"></td>
   <td height="50"> Raw data preprocessing - Alignment of RNA-Seq data</td>
</tr>
<tr>
   <td height="50">13:00 - 14:00</td>
   <td colspan="3" height="50"> Lunch break </td>
</tr>
<tr>
   <td height="50">14:00 - 14:45</td>
   <td height="50">Lecture </td>
   <td height="50"></td>
   <td height="50">Transcriptome Assembly and Quantifying Transcripts (Tuxedo Suite) </td>
</tr>
<tr>
   <td height="50">14:45 - 15:30</td>
   <td height="50">Lecture </td>
   <td height="50"></td>
   <td height="50">Alternative splicing events – Fusion genes</td>
</tr>
<tr>
   <td height="50">15:30 - 16:00</td>
   <td colspan="3" height="50"> Coffe break </td>
</tr>
<tr>
   <td height="50">16:00 - 17:45</td>
   <td height="50">Practical </td>
   <td height="50"></td>
   <td height="50">Transcriptome Assembly – Quantifying Transcripts –  Alternative splicing events</td>
</tr>
<tr>
   <td height="50">17:45 - 18:30</td>
   <td height="50">Talk </td>
   <td height="50"></td>
   <td height="50">“About Single-cell RNA-Seq”</td>
</tr>
<tr>
<td colspan="4"><h3>Friday - 29 Sep 2017</h3></td>
</tr>
<tr>
   <td height="50">09:15 - 09:30</td>
   <td colspan="3" height="50"> Wrap up </td>
</tr>
<tr>
<td height="50">09:30 - 10:30</td>
<td height="50">Lecture</td>
<td height="50"></td>
<td height="50">Differential Expression Analysis (Tuxedo Suite)</td>
</tr>
<tr>
   <td height="50">10:30 - 11:00</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
<td height="50">11:00 - 13:00</td>
<td height="50">Practical</td>
<td height="50"></td>
<td height="50">Differential Expression Analysis - Fusion genes</td>
</tr>
<tr>
   <td height="50">13:00 - 14:00</td>
   <td colspan="3" height="50"> Lunch break </td>
</tr>
<tr>
<td height="50">14:00 - 15:00</td>
<td height="50">Lecture</td>
<td height="50"></td>
<td height="50">Downstream analysis: enrichment function analysis and beyond</td>
</tr>
<tr>
<td height="50">15:00 - 16:00</td>
<td height="50">Practical</td>
<td height="50"></td>
<td height="50">Downstream analysis</td>
</tr>
<tr>
   <td height="50">16:00 - 16:30</td>
   <td colspan="3" height="50">  Final wrap-up and feedback </td>
</tr>
</table>

<br>

<h3>ELIXIR-IIB member institutions</h3>
<ul>
   <li> CNR (ELIXIR-IIB coordinator)</li>
   <li> CRS4</li>
   <li> CINECA</li>
   <li> Fondazione Edmund Mach, Trento</li>
   <li> INFN</li>
   <li> GARR</li>
   <li> Sapienza Università di Roma</li>
   <li> Università di Bari</li>
   <li> Università di Bologna</li>
   <li> Università di Firenze</li>
   <li> Università di Milano</li>
   <li> Università di Milano Bicocca</li>
   <li> Università di Padova</li>
   <li> Università di Parma</li>
   <li> Università di Roma "Tor Vergata"</li>
   <li> Università di Salerno</li>
   <li> Università della Tuscia </li>
</ul>


