---
layout: post
title:  "Training course on Machine Learning for Biologists <font color='black'>[closed]</font>"
date_start:   2017-09-04
date_end:   2017-09-07
venue: San Michele all'Adige (TN), Italy
description: The aim of the course is to provide a practical introduction to the analysis of "omics" data. Topics will range from data visualization/exploration to univariate/multivariate analysis and machine learning. Practical examples and applications will be illustrated by using R and Python.


---

<table border="0" width="600">
  <tr>
    <td><img src="../../../img/logo_elixir_italy.jpg" height="100"></td>
    <td><img src="../../../img/Logo_fbk.jpeg" height="80" width="170"></td>
    <td><img src="../../../img/Logo_fem.png" height="80" width="150"></td>
  </tr>
</table>

<br>

**IMPORTANT DATES** for this Training course:
  - Deadline for applications: 21 July 2017
  - Course date: 4-7 September 2017<br>
Chosen participants will be notified by 25 July 2017. A maximum of **24** candidates will be accepted in the course.
  <br>
  
### Course Description
The aim of the course is to provide a practical introduction to the analysis of "omics" data.
Topics will range from data visualization/exploration to univariate/multivariate analysis and machine learning. 
Practical examples and applications will be illustrated by using R and Python.

Course Milestones:
- Data exploration and visualization
- Univariate/Multivariate analysis
- Introduction to machine learning: classifiers, performance measures, diagnostics
- Machine learning tools for the analysis of Gene Expression data
- The Data Analysis Plan (DAP) - intro to unbiased pipelines for (binary) classification
- Performance measures and diagnostic plots - Accuracy, MCC, Stability: theory and graphics
- Differential network analysis -- co-expression networks, graph comparison, community detection: theory and examples in R/Python, visualization by the igraph library and use of the ReNette web interface
- Basic application of ML to gene prediction
<br>

### Target audience
A maximum of **24** students will be accepted. Computational biologists, bioinformaticians, biological data analysts.
<br>

### Learning objectives/outcomes
Acquisition of a working knowledge about running a full classification/profiling pipeline on omics data (e.g., gene expression). After this course participants should be able to:
<ul><li>Develop and implement visualization/exploration tools</li>
<li>Understand advantages and limitations of univariate and multivariate data analysis strategies in the context of omics research</li>
<li>Understand principles and applications of ML</li>
<li>Implement reproducible workflows in data analysis</li>
</ul>

### Course prerequisites
  basics of R, Python, \*nix (Unix or Linux) shell, basic statistical knowledge, basic knowledge of linear algebra
<br>
<br>
**FEE:**
The fee will cover attendance to lectures and practicals, coffee breaks, lunches, the welcome aperitivo on Monday September 4th. The participants are expected to pay their own travel and accommodation costs.

  - Candidates from a bioinformatics lab from one of the ELIXIR-IIB member institutions (see the list at the bottom): **120 euros**.
  - All the other candidates: **160 euros**
  
<br>
### [Application Form](https://goo.gl/forms/q5kN1QS4vgxvhrxA2)
<br>

**VENUE:**
 Fondazione Edmund Mach, Palazzo della Ricerca e della Conoscenza, Via E. Mach 1, San Michele all'Adige (TN), Italy, (zip code 38010).<br> General Info: [http://www.fmach.it/eng/CRI/general-info](http://www.fmach.it/eng/CRI/general-info)<br>
The campus: [http://www.fmach.it/eng/About-us/Campus](http://www.fmach.it/eng/About-us/Campus)

To get in:
   - By train: Trenitalia (national railways) stops at Trento or Mezzocorona and then [local train](http://www.ttesercizio.it/Treno/) to San Michele all'Adige
   - By car: The venue is 5' drive from the highway A22 (autostrada del Brennero), exit of "San Michele all'Adige/Mezzocorona".
   - By fly: From Verona airport please use the [shuttle bus](http://www.aeroportoverona.it/en/aerobus_t2) to Verona train station and then the train towards Trento.

SUGGESTED ACCOMMODATIONS:
A limited number of rooms will be available at the venue (costs 20 EUR/night). Please contact the local organizer (<alessandro.cestaro@fmach.it>). 
Here is some information about [hotels](../../../docs/Info_alberghi_san_michele_trento.pdf) close to San Michele all'Adige.

   
### Instructors and helpers
- [**Giuseppe Jurman**](http://mpba.fbk.eu/people/profile/jurman) - Fondazione Bruno Kessler, Trento, IT
- [**Marco Chierici**](http://mpba.fbk.eu/people/profile/chierici) - Fondazione Bruno Kessler, Trento, IT
- [**Davide Albanese**](../../../instructors/davide_albanese.html) - Fondazione E. Mach, Trento, IT
- **Pietro Franceschi** - Fondazione E. Mach, Trento, IT
- [**Marco Moretto**](../../../instructors/marco_moretto.html) - Fondazione E. Mach, Trento, IT
- [**Paolo Sonego**](../../../instructors/paolo_sonego.html) - Fondazione E. Mach, Trento, IT
- **Samantha Riccadonna** - Fondazione E. Mach, Trento, IT
- **Andrea Cattani**  - Fondazione E. Mach, Trento, IT
<br>

### Organisers
- Alessandro Cestaro (Local Organizer, Fondazione E. Mach, Trento, IT )
- Vincenza Colonna (ELIXIR-IIB Training Coordinator Deputy, CNR, IT)
<br>

### Programme

Course materials are available from the github repository:<br> [https://github.com/ELIXIR-IIB-training/MLB2017](https://github.com/ELIXIR-IIB-training/MLB2017)


<table border="1">
<tr>
   <td colspan="4"><h3>Monday 04 September 2017 - Introduction </h3></td>
</tr>
<tr>
   <td>14:00-15:30</td>
   <td>Course opening</td>
   <td colspan="2">Participants’ self-presentations</td>
</tr>
<tr>
   <td>15:30-16:15</td>
   <td>Plenary lecture </td>
   <td >P. Franceschi, S. Riccadonna</td>
   <td>Introduction to "omics" data. Principles of data exploration and  analysis</td>
</tr>
<tr>
   <td>16:15-16:45</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>16.45-19:00</td>
   <td>Practical</td>
   <td>P. Franceschi, S. Riccadonna</td>
   <td>Introduction to "omics" data. Principles of data exploration and  analysis </td>
</tr>
<tr>
   <td>19:30-21:00</td>
   <td colspan="3">Welcome aperitivo at Cantina storica Istituto Agrario San Michele</td>
</tr>
<tr>
   <td colspan="4"><h3>Tuesday  05 September 2017 - Machine Learning </h3></td>
</tr>
<tr>
   <td>09:30-09:40</td>
   <td>Previously On</td>
   <td colspan="2">Recap of previous lessons by participants</td>
</tr>
<tr>
  <td>09:40-10:30</td>
  <td>Lecture</td>
  <td>D. Albanese, P. Franceschi, S. Riccadonna</td>
  <td>Univariate and Multivariate analysis</td>
</tr>
<tr>
   <td>10:30-11:00</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>11:00-13:00</td>
   <td>Practical</td>
   <td>D. Albanese, P. Franceschi, S. Riccadonna</td>
   <td>Univariate and Multivariate analysis. Practical session with R</td>
</tr>
<tr>
   <td>13:00-14:30</td>
   <td colspan="3"> Lunch </td>
</tr>
<tr>
   <td>14:30-16:15</td>
   <td>Lecture</td>
   <td>D. Albanese, P. Franceschi, S. Ricadonna</td>
   <td>Machine Learning: introduction and applications to biological data. Classification basics, model selection and prediction</td>
</tr>
<tr>
<td>16:15-16:45</td>
<td colspan="3">Coffee Break</td>
<tr>
</tr>
   <td>16:45-18:30</td>
   <td>Practical</td>
   <td>D. Albanese, P. Franceschi, S. Riccadonna</td>
   <td>Performance measures and diagnostic plots</td>
</tr>

<tr>
   <td colspan="4"><h3>Wednesday  06 September 2017</h3></td>
</tr>
<tr>
   <td>09:30-09:40</td>
   <td>Previously On</td>
   <td colspan="2">Recap of previous lessons by participants</td>
</tr>
<tr>
  <td>09:40-10:30</td>
  <td>Lecture</td>
  <td>P. Sonego, S. Riccadonna </td>
  <td>Analyzing Gene Expression Data</td>
</tr>
<tr>
   <td>10:30-11:00</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>11:00-13:00</td>
   <td>Practical</td>
   <td>P. Sonego, S. Riccadonna </td>
   <td>Analyzing Gene Expression Data</td>
</tr>
<tr>
   <td>13:00-14:30</td>
   <td colspan="3"> Lunch </td>
</tr>
<tr>
   <td>14:30-16:15</td>
   <td> Lecture </td>
   <td>M. Chierici, G. Jurman </td>
   <td>The Data Analysis Plan (DAP) - intro to unbiased pipelines for (binary) classification</td>
</tr>
<tr>
<td>16:15-16:45</td>
<td colspan="3">Coffee Break</td>
<tr>
</tr>
   <td>16:45-18:30</td>
   <td>Practical</td>
   <td>M. Chierici, G. Jurman</td>
   <td>Implementation of a basic DAP in Python (Scikit-Learn) with feature ranking and classification</td>
</tr>
<tr>
<td colspan="4"><h3>Thursday 07 September 2017</h3></td>
</tr>
<tr>
   <td>09:15-09:25</td>
   <td>Previously On</td>
   <td colspan="2">Recap of previous lessons by participants</td>
</tr>
<tr>
   <td>09:25-10:00</td>
   <td>Lecture</td>
   <td>M. Moretto, A. Cestaro</td>
   <td>Gene prediction methods as an example of ML on genomic data</td>
</tr>
<tr>
<td>10:00-10:30</td>
<td colspan="3">Coffee Break</td>
</tr>
<tr>
<td>10:30-12:30</td>
<td>Practical</td>
<td>M. Moretto, A. Cestaro</td>
 <td>Training a gene prediciton method</td>
</tr>
<tr>
<td>12:30-13:00</td>
<td colspan="3">Wrap-up and feedback</td>
</tr>
</table>

<br>

<h3>ELIXIR-IIB member institutions</h3>

<ul>
   <li> CNR (ELIXIR-IIB coordinator)</li>
   <li> CRS4</li>
   <li> CINECA</li>
   <li> Fondazione Edmund Mach, Trento</li>
   <li> INFN</li>
   <li> GARR</li>
   <li> Sapienza Università di Roma</li>
   <li> Università di Bari</li>
   <li> Università di Bologna</li>
   <li> Università di Firenze</li>
   <li> Università di Milano</li>
   <li> Università di Milano Bicocca</li>
   <li> Università di Padova</li>
   <li> Università di Parma</li>
   <li> Università di Roma "Tor Vergata"</li>
   <li> Università di Salerno</li>
   <li> Università della Tuscia</li>
</ul>
