---
layout: post
title:  "Population Genomics: background and tools <font color='green'>[OPEN]</font>"
date_start:   2018-04-21
date_end:   2018-04-27
venue: Napoli, Italy
description: Study methods in population genomics have been profoundly reshaped in the last few years thanks to the growing availability of complete genomic sequences at population level. The rapid and recent growth of data and methods calls for new approaches to become routine in evolutionary genomics laboratories. The objective of this Practical Course is to give an overview of state of the art methods in population genomics combining lecturing from outstanding experienced population geneticists and software developers. All conceptual innovation will be presented in lectures and applied in practice both individually and in group work.

---
<table border="0">
<tr>
	<td><img src="../../../img/logo_elixir_popgen.2018.png">
	</td>	
</tr>
</table>
<br>
<table border="0" width="600">
   <tr>
     <td><img src="../../../img/logos/IGB-logo.jpg" alt="cnr" height="150" width="350"></td>
      <td><img src="../../../img/elixir_ita_logo.png" alt="yay" height="150" width="300"></td>
      <td width="100"></td>
    <td><img src="../../../img/tw.png" alt="yay" height="100" width="150"></td>
    <td><b>#popgenELIXIR</b></td>
   </tr>
</table>
<br>

Study methods in population genomics have been profoundly reshaped in the last few years thanks to the growing availability of complete genomic sequences at population level. The rapid and recent growth of data and methods calls for new approaches to become routine in evolutionary genomics laboratories. The objective of this Practical Course is to give an overview of state of the art methods in population genomics combining lecturing from outstanding experienced population geneticists and software developers. All conceptual innovation will be presented in lectures and applied in practice both individually and in group work.

Practicals includes computer exercises supervised by lecturers and training assistants. <br>
After attending the course participants will be aware of up-to-date concepts in population genetics, capable of running analyses using software based on whole genome data sequences and able to deal with basic aspects of any population genomics project.<br>

This Practical Course aims at evolutionary biologists who already have bioinformatics skills. PhD students and Post-Doc researchers will benefit the most out of this course, but applications from all candidates will be evaluated in their context.<br>
Participants will need to bring their own laptop in order to connect to a server to run the practical exercise. 

## Important Dates 
<b>Deadline for applications:</b> 5 February 2018 <br>
<b>Chosen participants will be notified by:</b> 28 February 2018<br> 
<b>Payment deadline:</b> 10 March 2018<br>
<b>Course date:</b> 21-27 April 2018<br>

## Venue
Biblioteca <br> 
Consiglio Nazionale delle Ricerche <br> 
Istituto di Genetica e Biofisica "Adriano Buzzati-Traverso" <br> 
via Pietro Castellino 111 <br> 
80131 Napoli - IT <br> 
[www.igb.cnr.it](www.igb.cnr.it)

## Registration fee: 500€
Registration fee includes course material, coffee breaks and lunches. A maximum of **28** candidates will be accepted in the course. Priority will be given to candidates from ELIXIR-IIB member institutions (see the list at the bottom) and ELIXIR nodes. Incomplete applications or applications for only parts of the course will not be accepted.<br>
Abstract (max 200 words) for poster presentation must be submitted through the application form.
<br>
<br>

## [Application form](https://goo.gl/forms/dDQLyAB4cX1iRNMb2)
<br> 

## Speakers 
<table border="1">
 <tr height="200">
   <td width="210" height="200"> 
      <a href="http://www.eva.mpg.de/genetics/genetic-diversity-and-selection/overview.html">Aida Andrés</a> <br>
      UCL Genetics Institute,<br>London,<br>UK
      <br>
      <img src="../../../img/speakers/AidaAndres.jpg" alt="yay" width="150" height="150">
    </td>
     <td width="210">  
      <a href="http://www.zoo.cam.ac.uk/directory/andrea-manica">Andrea Manica </a><br>
     University of Cambridge,<br>Cambridge,<br>UK <br>
     <img src="../../../img/speakers/AndreaManica.jpeg" alt="yay" width="150" height="150">
    </td>
     <td width="210"> 
      <a href="https://blogs.cornell.edu/clarklabblog/">Andrew Clark </a><br>
      Cornell University,<br>New York,<br> USA<br>
      <img src="../../../img/speakers/AndrewClarkOK.png" alt="yay" width="150" height="150">
    </td>
   </tr>
   
   <tr height="200">
    <td> 
      <a href="http://www.homepages.ucl.ac.uk/~ucbtgrh/"> Garrett Hellenthal</a><br>
       University College London,<br> UK<br>
       <img src="../../../img/speakers/GarrettHellenthalOK.jpg" alt="yay" width="150">
    </td>
    <td> 
      <a href="https://idamoltke.wordpress.com/">Ida Moltke</a><br>
      University of Copenhagen, Denmark<br>
      <img src="../../../img/speakers/IdaMoltkeOK.jpg" alt="yay" width="150">
    </td>
    <td> 
      <a href="http://geogenetics.ku.dk/staff/?pure=en%2Fpersons%2Fmartin-sikora(ca4b071d-000d-4aaf-badb-67d9b017ec38)%2Fpublications.html">Martin Sikora</a><br>
       Natural History Museum of Denmark, Denmark<br>
       <img src="../../../img/speakers/MartinSikoraOK.jpg" alt="yay" width="150">
    </td>
    </tr> 
    <tr height="200"> 
    <td> 
      <a href="http://ua.unige.ch/en/personne/mathiascurrat/">Mathias Currat </a><br>
      Université de Genève,<br> Switzerland<br>
      <img src="../../../img/speakers/MathiasCurratOK.jpg" alt="yay" width="150">
    </td>
    <td> 
      <a href="http://funpopgen.unige.ch/">Olivier Delaneau</a><br>
     Université de Genève,<br> Switzerland<br>
      <img src="../../../img/speakers/OlivierDelaneauOK.jpg" alt="yay" width="150">
    </td>
    <td> 
      <a href="https://www.westminster.ac.uk/about-us/our-people/directory/gerbault-pascale">Pascale Gerbault</a><br>
      University of Westminster,<br> UK<br>
      <img src="../../../img/speakers/PascaleGerbaultOK.jpg" alt="yay" width="150">
    </td>
   </tr>
</table>
<br>

## Programme

<table border="1">
<tr>
   <td colspan="4"><h3>Saturday 21 April 2018</h3></td>
</tr>
<tr>
   <td>9:00-9:30</td>
   <td colspan="3">Coffee and registration</td>
</tr>
<tr>
   <td>9:30-11:00</td>
   <td>Welcome session</td>
   <td>Organizers, Trainers and Participants</td>
   <td>Introduction to the course, presentation of the course website and repository with all the material for the course. Trainers and participants presentation: current position in science, course motivations, objectives and expectations.</td>
</tr>
<tr>
   <td>11:00-12:00</td>
   <td>Practical</td>
   <td>Allegra Via</td>
   <td>Linux recap - Assessment of participants for basic knowledge of linux necessary for the following sessions. Quick recapitulation of linux based on the assessment to ensure all participants are at the same level.</td>
</tr>
<tr>
   <td>12:00-13:00</td>
   <td>Keynote lecture</td>
   <td>Andrew Clark</td>
   <td>Population Genomics: where are we going? - Overview of major achievements and current goals of population genomics.</td>
</tr>
<tr>
   <td>13:00-14:15</td>
   <td colspan="3">Lunch and coffee</td>
</tr>
<tr>
   <td>14:15-15:00</td>
   <td>Lecture</td>
   <td>Olivier Delaneau</td>
   <td>Data QC and Exploratory Data Analysis - Basic operations to perform on genome sequences data sets before applying population genomic analyses.</td>
</tr>
<tr>
   <td>15:00-17:00</td>
   <td>Practical</td>
   <td>Olivier Delaneau</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on clustering methods to detect sample substructure (e.g. principal component analyses).</td>
</tr>
<tr>
   <td colspan="4"><h3>Sunday 22 April 2018</h3></td>
</tr>
<tr>
   <td>9:00-9:30</td>
   <td>Recap session</td>
   <td>Participants and Trainers</td>
   <td>Quick recapitulation of concepts from the previous day</td>
</tr>
<tr>
   <td>09:30-10:15</td>
   <td>Lecture</td>
   <td>Olivier Delaneau</td>
   <td>Haplotype phasing and imputation - Attribution of alleles on maternal and paternal chromosomes to determine haplotypes, statistical inference of unobserved genotypes using known haplotypes in a population.</td>
</tr>
<tr>
   <td>10:15-10:30</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>10:30-12:30</td>
   <td>Practical </td>
   <td>Olivier Delaneau</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on imputation and phasing using the software Impute, and Shapeit.</td>
</tr>
<tr>
   <td>12:30-13:30</td>
   <td colspan="3">Lunch in the park</td>
</tr>
<tr>
   <td>13:30-14:15</td>
   <td>Lecture</td>
   <td>Andrew Clark</td>
   <td>Forces shaping genetic diversity: recombination, mutation, selection, drift - An overview of basic concepts in population genomics in the light of the recent achievements and discoveries.</td>
</tr>
<tr>
   <td>14:15-16:30</td>
   <td>Practical</td>
   <td>Andrew Clark</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on classical and modern population genomics concepts. Coffee will be available during the practicals.</td>
</tr>
<tr>
   <td>16:30-17:00</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>17:00-18:00</td>
   <td>Lecture</td>
   <td>Andrew Clark</td>
   <td>Coalescence theory made fun and easy / Coalescent models describing genetic diversity in populations as consequence of their demographic history.</td>
</tr>
<tr>
   <td colspan="4"><h3>Monday 23 April 2018</h3></td>
</tr>
<tr>
   <td>9:00-9:30</td>
   <td>Recap session</td>
   <td>Participants and Trainers</td>
   <td>Quick recapitulation of concepts from the previous day</td>
</tr>
<tr>
   <td>09:30-10:15</td>
   <td>Lecture</td>
   <td>Ida Moltke</td>
   <td>Inference of relatedness and identity by descent sharing - Relatedness inference and identical by descent sharing.</td>
</tr>
<tr>
   <td>10:15-10:30</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>10:30-12:30</td>
   <td>Practical </td>
   <td>Ida Moltke</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on relatedness inference and identical by descent sharing.</td>
</tr>
<tr>
   <td>12:30-13:30</td>
   <td colspan="3">Lunch in the park</td>
</tr>
<tr>
   <td>13:30-15:30</td>
   <td>Practical</td>
   <td>Andrew Clark</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on genetic drift and coalescence: coalescent simulations made with pen and paper. Coffee will be available during the practicals.</td>
</tr>
<tr>
   <td>15:30-16:00</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>16:00-18:00</td>
   <td colspan="3">Poster session</td>
</tr>
<tr>
   <td colspan="4"><h3>Tuesday 24 April 2018</h3></td>
</tr>
<tr>
   <td>9:00-9:30</td>
   <td>Recap session</td>
   <td>Participants and Trainers</td>
   <td>Quick recapitulation of concepts from the previous day</td>
</tr>
<tr>
   <td>09:30-10:15</td>
   <td>Lecture</td>
   <td>Martin Sikora</td>
   <td>f3 and f4 statistics - Methods to detect population admixture from genomic data .</td>
</tr>
<tr>
   <td>10:15-10:30</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>10:30-12:30</td>
   <td>Practical </td>
   <td>Martin Sikora</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on population admixture using the software ADMIXTOOL. Possible extra topic: ROLLOFCoffee will be available during the practicals.</td>
</tr>
<tr>
   <td>12:30-13:30</td>
   <td colspan="3">Lunch in the park</td>
</tr>
<tr>
   <td>13:30-15:00</td>
   <td colspan="3">Poster session</td>
</tr>
<tr>
   <td>15:00-19:00</td>
   <td colspan="3">Walk in Naples</td>
</tr>
<tr>
   <td colspan="4"><h3>Wednesday 25 April 2018</h3></td>
</tr>
<tr>
   <td>9:00-9:30</td>
   <td>Recap session</td>
   <td>Participants and Trainers</td>
   <td>Quick recapitulation of concepts from the previous day</td>
</tr>
<tr>
   <td>09:30-10:15</td>
   <td>Lecture</td>
   <td>Garrett Hellenthal</td>
   <td>Clustering algorithms - Partitioning of human genetic variation into a small number of groups to infer population structure and assign individuals to hypothesized ancestral groups.</td>
</tr>
<tr>
   <td>10:15-10:30</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>10:30-12:30</td>
   <td>Practical </td>
   <td>Garrett Hellenthal</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on clustering using the software ADMIXTURE, CHROMOPAINTER/fineSTRUCTURE.</td>
</tr>
<tr>
   <td>12:30-13:30</td>
   <td colspan="3">Lunch in the park</td>
</tr>
<tr>
   <td>13:30-14:15</td>
   <td>Lecture</td>
   <td>Garrett Hellenthal</td>
   <td>Haplotype-based methods for demography - Reconstruction of population demography (admixture, population splits and size changes) from genetic data using haplotypes.</td>
</tr>
<tr>
   <td>14:15-16:30</td>
   <td>Practical</td>
   <td>Garrett Hellenthal</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on inferring population demography (admixture population splits and size changes) using the software MSMC/PSMC and GLOBETROTTER. Coffee will be available during the practicals.</td>
</tr>
<tr>
   <td colspan="4"><h3>Thursday 26 April 2018</h3></td>
</tr>
<tr>
   <td>9:00-9:30</td>
   <td>Recap session</td>
   <td>Participants and Trainers</td>
   <td>Quick recapitulation of concepts from the previous day</td>
</tr>
<tr>
   <td>09:30-10:15</td>
   <td>Lecture</td>
   <td>Mathias Currat</td>
   <td>Gene-genealogy methods for demography, Approximate Bayesian Computation - Reconstruction of population demography from genetic data using coalescent simulations and Approximate Bayesian Computation.</td>
</tr>
<tr>
   <td>10:15-10:30</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>10:30-12:30</td>
   <td>Practical </td>
   <td>Mathias Currat</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on ABC using the software fastSIMCOAL.</td>
</tr>
<tr>
   <td>12:30-13:30</td>
   <td colspan="3">Lunch in the park</td>
</tr>
<tr>
   <td>13:30-14:15</td>
   <td>Lecture</td>
   <td>Pascale Gerbault</td>
   <td>Types and measurements of natural selection - Mechanisms through which new advantageous genetic variants become fixed in populations and natural selection favours extreme phenotypes in changing environments.</td>
</tr>
<tr>
   <td>14:15-16:30</td>
   <td>Practical</td>
   <td>Pascale Gerbault</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on detecting natural selection. Coffee will be available during the practicals.</td>
</tr>
<tr>
   <td>16:30-17:00</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>17:00-18:00</td>
   <td>Keynote lecture</td>
   <td>Andrea Manica</td>
   <td>“Climate changes, demography and migrations in the context of human evolution” - How demography and natural selection can help understanding human diseases.</td>
</tr>
<tr>
   <td colspan="4"><h3>Friday 27 April 2018</h3></td>
</tr>
<tr>
   <td>9:00-9:30</td>
   <td>Recap session</td>
   <td>Participants and Trainers</td>
   <td>Quick recapitulation of concepts from the previous day</td>
</tr>
<tr>
   <td>09:30-10:15</td>
   <td>Lecture</td>
   <td>Andrea Manica</td>
   <td>Selection: phenotypes and genotypes / Selection scans and GWASs:examples and challenges.</td>
</tr>
<tr>
   <td>10:15-10:30</td>
   <td colspan="3">Coffee break</td>
</tr>
<tr>
   <td>10:30-12:30</td>
   <td>Practical </td>
   <td>Andrea Manica</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on genome-wide detection of natural selection with explicit drivers (phenotype or environment).</td>
</tr>
<tr>
   <td>12:30-13:30</td>
   <td colspan="3">Lunch in the park</td>
</tr>
<tr>
   <td>13:30-14:15</td>
   <td>Lecture</td>
   <td>Aida Andrés</td>
   <td>Species divergence: the case of our close relatives - Genetic evolution of humans and closely related species, an example of how natural selection has shaped the affinities and differences that exist among individuals, populations and species.</td>
</tr>
<tr>
   <td>14:15-16:30</td>
   <td>Practical</td>
   <td>Aida Andrés</td>
   <td>Practical demonstrations (estimated 45' within practicals). Individual exercises on species divergence. Coffee will be available during the practicals.</td>
</tr>
  </table>
<br>

## Transport 
#### HOW TO REACH NAPOLI

#### ARRIVING IN NAPLES BY TRAIN

The central train stations is "Napoli Centrale" or "Napoli Piazza Garibaldi" (the same location). To book a train there are two train companies: Trenitalia and Italo. The train station is connected to the city by the metro Linea1.

#### ARRIVING IN NAPLES BY PLANE

From the International Airport Napoli Capodichino there are two ways to reach the city centre or the train station:

Bus - There is a bus called Alibus that you can take at the airport. Tickets are available on board (4 euros, only cash) or at the newspaper/souvenir shop in the airport ground floor (3 euros) and are valid for metro as well. The bus has two stops: first Piazza Garibaldi i.e. by the central train station, and second is by the city harbour where ships for the gulf islands leave.

Taxi - Ask for “tariffa predeterminata” (fixed fare) if you are going to one of the zones included in the offer. For the city centre or the train station it costs 16 euros, for Vomero area 23 euros. It is mandatory for taxi drivers to apply the fixed fare if required. Ask for the cost before starting the ride.

#### HOW TO REACH THE BIBLIOTECA AT CNR

The CNR is right outside the exit Montedonzelli of the metro Linea1. Check this website for more details. Once at the CNR reception you will find indications for the course. To enter the building it is mandatory to sign a registry of presence at the reception. You will be asked to sign when you enter for the first time in the morning and when you leave in the evening. This is a security requirement.

## Contact
Email: <elixir.popgen.2018@gmail.com> <br>
Phone: +39 0816 132 254

## Organisers 

<table>
 <tr>
     <td width="150"> 
      <a href="https://www2.le.ac.uk/departments/health-sciences/people/staff-pages/research-and-teaching-staff-pages/chiara-batini">Chiara Batini </a><br>
     University of Leicester, UK <br>
     <img src="../../../img/organisers/ChiaraBatiniOK.jpg" alt="yay" height="150" width="150">
    </td>
     <td width="150"> 
      <a href="http://www.igb.cnr.it/popgenlab">Vincenza Colonna </a><br>
      Consiglio Nazionale delle Ricerche, Italy<br>
      <img src="../../../img/organisers/VincenzaColonnaOK.jpg" alt="yay" height="150" width="150">
    </td>
    <td width="150"> 
      <a href="https://allegravia.github.io/website/publications/">Allegra Via</a> <br>
      Consiglio Nazionale delle Ricerche, Italy  <br> 
      <img src="../../../img/organisers/AllegraVia.jpg" alt="yay" height="150" width="150">
    </td>
   </tr>   
  </table>

<br>

## Pompei - Ercolano


<table style="width:100%">
   <tr>
     <td><img src="../../../img/pompei1.jpg" alt="yay" height="300"></td>
     </tr>
</table>
<br>

[Pompei](https://en.wikipedia.org/wiki/Pompeii) and [Ercolano](https://en.wikipedia.org/wiki/Ercolano) are two famous archaeological sites that lie in the shadow of [mount Vesuvius](https://en.wikipedia.org/wiki/Mount_Vesuvius), the active volcano that dominates the area.

They were destroyed by pyroclastic flows during the massive eruption of Vesuvius in 79 AD.

Go to the central train station at Piazza Garibaldi. You'll find the Circumvesuviana (the train system that connects the Napoli and the towns and cities in the region around Vesuvius) on the lower level of the station. Buy tickets (about 3 euro each way) and take the Circumvesuviana to Pompei _Scavi_. Note that other regional trains go to the modern city of Pompei but you will need to walk about 25 minutes from that station to the archaeological site while the station Pompei Scavi is just a few minutes from the entrance to the park. A single cafeteria serves the site, so you may want to bring snacks. It can be hot and there is virtually no shade, so dress accordingly. Pompei is a vast ruin, and a walk through most of it should take at least three hours.

Ercolano lies on the same train line as Pompei, but is closer to Napoli. To get there, leave the Circumvesuviana at Ercolano Scavi, and walk down the hill towards the sea for about ten minutes. Ercolano is much smaller than Pompei, but what it lacks in size it makes up in the quality of its preservation. It is possible to visit both sites in one day, but you will probably want to start early.
<br>
<br>

<h3>ELIXIR-IIB member institutions</h3>

<ul>
   <li> CNR (ELIXIR-IIB coordinator)</li>
   <li> CRS4</li>
   <li> CINECA</li>
   <li> Fondazione Edmund Mach, Trento</li>
   <li> INFN</li>
   <li> GARR</li>
   <li> Sapienza Università di Roma</li>
   <li> Università di Bari</li>
   <li> Università di Bologna</li>
   <li> Università di Firenze</li>
   <li> Università di Milano</li>
   <li> Università di Milano Bicocca</li>
   <li> Università di Padova</li>
   <li> Università di Parma</li>
   <li> Università di Roma "Tor Vergata"</li>
   <li> Università di Salerno</li>
   <li> Università della Tuscia</li>
</ul>
