---
layout: post
title:  "Gene Ontology and Bioinformatics Data Resources Workshop: data annotation and functional analysis tools. <font color='red' size='3'>[cancelled, to be rescheduled]</font>"
date_start:   2020-05-06
date_end:   2020-05-07
venue: Rome, Italy
description: Two-day workshop to boost your understanding of bioinformatics resources and their use - you will learn about the key data resources available and how to use functional analysis tools.

---


<table border="0" align="center">
	<tr>
		<td><a href="http://elixir-italy.org"><img src="../../../img/logo_iib.png" height="100"></a></td>
		<td width="50"></td>
		<td><a href="https://en.uniroma2.it/"><img src="../../../img/Logo_uni_torvergata.png" height="110"></a></td>
		<td width="50"></td>
		<td><a href="https://www.ucl.ac.uk/"><img src="../../../img/logo_UCL.jpeg" height="100"></a></td>
	</tr>
</table>
<br>
<br>
<br>

**<font color='red'>WARNING: due to the public health emergency, the course has been cancelled and will be rescheduled to a later date.</font>**
<br>
<br>


### Important Dates
- Deadline for applications: **31 March 2020** 
- Chosen participants will be notified by: 10 April 2020
- Course date: **6-7 May 2020**
<br>
<br>

### Venue
University of Rome Tor Vergata<br>
Aula 17, Department of Biology, Rome. [**(link to the map)**](https://www.google.com/maps/place/Macroarea+di+Scienze+M.F.N.+-+Universit%C3%A0+di+Roma+Tor+Vergata/@41.8541,12.6032513,17z/data=!4m12!1m6!3m5!1s0x132587f9328c769b:0x1391be005eeb11f9!2sMacroarea+di+Scienze+M.F.N.+-+Universit%C3%A0+di+Roma+Tor+Vergata!8m2!3d41.8541!4d12.60544!3m4!1s0x132587f9328c769b:0x1391be005eeb11f9!8m2!3d41.8541!4d12.60544)<br>
Via della Ricerca Scientifica<br>
00133 Rome, Italy
<br>
<br>

### Fee 
The course does not include a fee, but the participants are expected to pay their own travel and accomodation costs (if any).
<br>
<br>

### Selection
A maximum of **30 candidates** will be selected on a first-come-first-served basis. Chosen participants will be notified by 10th April 2020. 

**Cancellation policy:** Attendance is limited to 30 participants. We expect many more applications. Accepted participants commit to attend the course for its whole duration. Failure to attend training sessions is disruptive. Moreover, it blocks other candidates from participating. Therefore, a cancellation policy is in place so that only written requests presented 10 days in advance relatively to the course starting date are accepted.
<br>
<br>

### Instructors
- [**Ruth Lovering**]() -  University College of London, UCL, UK
- [**Luana Licata**](https://elixir-iib-training.github.io/website/instructors/luana_licata.html) - University of Rome Tor Vergata, ELIXIR-IT
- [**Ines Pineda-Torra**]() - University College of London, UCL, UK
<br>
<br>

### Organisers
- [**Ruth Lovering**]() -  University College of London, UCL, UK
- [**Luana Licata**](https://elixir-iib-training.github.io/website/instructors/luana_licata.html) - University of Rome Tor Vergata, ELIXIR-IT (Local Organiser)
<br>
<br>

### Contact
For all kinds of queries, please contact the local organiser Luana Licata at: <luana.licata@uniroma2.it>. For questions about course content, please email the scientific organisers <luana.licata@uniroma2.it> or <r.lovering@ucl.ac.uk>.
<br>
<br>

### Course Description
The course will provide information about Gene Ontology (GO) and molecular interaction resources and practical demonstrations of a number of freely available high-throughput analysis and network analysis tools that can be used for the analysis of large datasets, such as those derived from proteomic or transcriptomic methodologies.
<br>
<br>

### Target audience
The workshop is open to all biological or biomedical PhD and Post-Doc research scientists, as well as team leaders and PIs. You don't need to have any previous knowledge of the tools that will be presented at the workshop. Attendees may bring in their own datasets for high-throughput analysis, although example datasets will be available.
<br>
<br>

### Aims of the workshop
-	To provide information about Gene Ontology (GO) and molecular interaction resources 
-	To provide practical demonstrations of a number of freely available high-throughput analysis and network analysis tools that can be used for the analysis of large datasets, such as those derived from proteomic or transcriptomic methodologies.<br>
Resources covered:
-	Identifier mapping
-	Gene Ontology
-	Functional analysis tools
-	Other ontologies 
-	Protein-Protein interaction data 
-	Directed molecular interactions
-	Cytoscape
-	Ensembl
-	Data mining: Biomart
<br>
<br>

### Learning Outcomes
By the end of this course, the participants will be able to:
- Identify and access public data resources and tools for data integration, high-throughput analysis, functional analysis and network analysis
- Exploit the information available in the Gene Ontology (GO) to identify key cellular processes associated with high-throughput datasets
- Interpret the information given in different molecular interaction and signaling databases
- Compare different sources of protein-protein interaction data and causal interaction data
- Represent networks using Cytoscape
- Identify tools used for network analysis and functional analysis
<br>
<br>

### Course prerequisites
Participants can use the computer available in the meeting room however if they prefer, they can bring their own laptop and install before the meeting few specific software.  Materials will be provided in advance of the workshop by the organizers. 
<br>
<br>

### [Application Form - Registration](https://docs.google.com/forms/d/e/1FAIpQLSeRrEW8ScR_9iEwa4hSNu6ko45bqbHk5RhC0z67oWWe3xtNAg/viewform)
<br>
<br>

### Preliminary Programme
<table border="1" width="700">
<tr>
   <td colspan="4"><h3>Wednesday - 6 May</h3></td>
</tr>
<tr>
   <td height="50" width="50">09:00 - 09:15</td>
   <td height="50">lecture</td>
   <td height="50"> What do you know about bioinformatics databases? Small group discussions and feedback to attendees</td>
</tr>
<tr>
   <td height="50">09:15 - 09:45</td>
   <td height="50">Lecture</td>
   <td height="50">Identifier mapping, HGNC/UniProt/others</td>
</tr>
<tr>
   <td height="50">09:45 - 10:45</td>
   <td height="50">Lecture</td>
   <td height="50">Gene Ontology. Examples of data captured</td>
</tr>
<tr>
   <td height="50">10:45 - 11:15</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:15 - 11:45</td>
 	<td height="50">Exercise</td>
   <td height="50">QuickGO browser</td>
</tr>
<tr>
   <td height="50">11:45 - 12:45</td>
  	<td height="50">Lecture and Exercise</td>
   <td height="50">GO-CAM and annotation extensions</td>
</tr>
<tr>
   <td height="50">12:45 - 13:45</td>
   <td colspan="3" height="50">Lunch break</td>
</tr>
<tr>
   <td height="50">13:45 - 14:15</td>
   <td height="50">Lecture</td>
   <td height="50">Functional analysis tools. How to cite data/tools in papers, including date of files/analysis. Checking the date the tools updated their annotation/ontology files</td>
</tr>
<tr>
   <td height="50">14:15 - 15:45</td>
    <td height="50">Lecture and Exercise</td>
   <td height="50">Transcriptomic data analysis by using variety of tools (TBC) g:Profiler,WebGestalt,DAVID,VLAD</td>
</tr>
<tr>
   <td height="50">15:45 - 16:15</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">16:15 - 17:30</td>
    <td height="50">Lecture and Exercise</td>
   <td height="50">Ontology Lookup Service</td>
</tr>
<tr>
   <td colspan="4"><h3>Thursday - 7 May</h3></td>
</tr>
<tr>
   <td height="50">09:00 - 10:30</td>
   <td height="50">Lecture and Exercise</td>
   <td height="50">Protein-Protein Interactions and Directed Molecular Interactions</td>
</tr>
<tr>
   <td height="50">10:30 - 11:00</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">11:00 - 13:00</td>
   <td height="50">Lecture and Exercise</td>
   <td height="50">Cytoscape, GOlorize, BinGO</td>  
</tr>
<tr>
   <td height="50">13:00 - 14:00</td>
   <td colspan="3" height="50">Lunch break</td>
</tr>
<tr>
   <td height="50">14:00 - 15:00</td>
   <td height="50">Exercise</td>
   <td height="50">Continue with Cytoscape</td>
</tr>
<tr>
   <td height="50">15:00 - 15:45</td>
   <td height="50">Lecture and Exercise</td>
   <td height="50">Ensembl</td>
</tr>
<tr>
   <td height="50">15:45 - 16:15</td>
   <td colspan="3" height="50">Coffee break</td>
</tr>
<tr>
   <td height="50">16:15 - 17:30</td>
   <td height="50">Lecture and Exercise</td>
   <td height="50">Data mining: Biomart</td>
</tr>
</table>
<br>
<br>
<br>
<br>




<h3>ELIXIR-IIB member institutes</h3>
<ol>
   <li> <b>CNR, National Research Council </b> (Lead Institute)</li>
   <li> CRS4</li>
   <li> CINECA</li>
   <li> Edmund Mach Foundation, Trento</li>
   <li> ENEA</li>
   <li> Fondazione Telethon</li> 
   <li> INFN</li>
   <li> Istituto Superiore di Sanit√† (ISS)</li> 
   <li> GARR</li>
   <li> Stazione Zoologica Anton Dohrn, Napoli</li>
   <li> University of Roma "Sapienza"</li>
   <li> University of Bari</li>
   <li> University of Bologna</li>
   <li> University of Firenze</li>
   <li> University of Milano</li>
   <li> University of Milano Bicocca</li>
   <li> University of Napoli</li>
   <li> University of Padova</li>
   <li> University of Parma</li>
   <li> University of Roma "Tor Vergata"</li>
   <li> University of Salerno</li>
   <li> University of Torino</li>
   <li> University of Tuscia </li>
</ol>
<br>
<br>
