---
layout: post
title: "Training Course in Computational Methods for Epitrascriptomics"
date_start: 2023-04-26
date_end: 2023-04-28   
venue: Bari, Italy
description: This course will provide a foundation on the experimental planning, analytical strategies and computational approaches for applying Nanopore direct RNA Sequencing to profile RNA modifications. Through a combination of theoretical lectures and practical exercises, participants will learn the fundamental concepts and bioinformatics skills that will allow them to start using Nanopore sequencing data for epitranscriptome analysis.

---


<table border="0">
<tr>
	<td>
	<img src="../../../img/Logo_Computational_Methods_Bari.png" height="380" width="800">
	</td>	
</tr>	
</table>


<br>
<br>

## Important Dates 
- Deadline for applications: **15 April, 2023**
- Course date: **26-28 April, 2023** 
<br>
<br>

### Venue
Area della Ricerca del CNR di Bari<br>
[Dove Siamo](https://www.ba.cnr.it/dovesiamo.html)<br>
Via Amendola 122/o<br>
70126 Bari, Italy
<br>
<br>

### Fee
The course is free of charge (coffee breaks and a lunch are offered).
Travel, accommodation and additional meals are not included.

<br>
<br>

### Registration 

# [Application Form](https://forms.gle/N7KAFQCTuqcwRLCW7)
<br>

A maximum of 15 candidates will be accepted for the course. Course selection will be done according to the needs for the course as emerging from the application form. Priority will be given to candidates from ELIXIR-IT member institutions (see the [list](https://elixir-italy.org/about/)) and ELIXIR nodes. 
Acceptance will be notified soon after the application deadline. 
 
<br>
<br>
### Organisers
- **Ernesto Picardi** (UNIBA, IT)
- **Francesca De Leo** (CNR-IBIOM, IT)
- **Francesco Nicassio** (IIT, IT)
- **Tommaso Leonardi** (IIT, IT)
- **Bruno Fosso** (UNIBA, IT)
- **Graziano Pesole** (UNIBA, IT)


### Instructors

- **Adriano Fonzino** (UNIBA, IT)
- **Claudio Lo Giudice** (UNIBA, IT)
- **Alessandro Silvestris** (UNIBA, IT)
- **Francesco Nicassio** (IIT, IT)
- **Ernesto Picardi** (UNIBA, IT)
- **Tommaso Leonardi** (IIT, IT)
- **Camilla Ugolini** (IIT, IT)
- **Logan Mulroney** (IIT, IT)

<br>
<br>

### Helpers

- **Pietro Dâ€™Addabbo** (UNIBA, IT)
- **Chiara Bruno** (CNR, IT)

### Speakers

- **Eva Novoa** (CRG, ES)
- **Mattia Pelizzola** (IIT, IT)

<br>
<br>

### Course Description
Epitranscriptome refers to all chemical modifications affecting cellular RNAs. Although more than 170 different types have been identified up to now, their punctual detection is yet challenging. Profiling epitranscriptome modifications require ad hoc sequencing technologies as well as dedicated computational workflows. While Illumina RNAseq is the de facto sequencing methodology for unveiling RNA editing events (not transient modifications), Nanopore direct RNA Sequencing has recently become one of the most powerful techniques to profile the whole repertoire of RNA modifications. Its capacity to directly sequence full-length, native RNA molecules without the need for retrotranscription or amplification offers remarkable advantages compared to other technologies based on short-read sequencing. However, at the same time it poses various experimental and analytical challenges.
This course will provide a foundation on the experimental planning, analytical strategies and computational approaches for applying Nanopore direct RNA Sequencing to profile RNA modifications. Through a combination of theoretical lectures and practical exercises, participants will learn the fundamental concepts and bioinformatics skills that will allow them to start using Nanopore sequencing data for epitranscriptome analysis. Additionally, the course will also cover theoretical aspects of RNA editing as well as practical sessions focused on its detection in human/mouse transcriptomes by Illumina RNAseq data. Participants will acquire specific skills in command-line tools (REDItools) to call, annotate (REDIportal) and filter RNA variants.
You can find here the complete [program](https://drive.google.com/file/d/1rMYXnNN5ddKwOHNaFsy4Nn7w0q5rKcxH/view?usp=share_link). 

<br>
<br>

 Prerequisites
### Target audience and prerequisites
Lectures are intended for early/mid-career stage researchers with a solid understanding of molecular biology and RNA sequencing, but no prior experience with Nanopore sequencing is required. In order to follow and independently carry out the practical exercises participants will need basic bioinformatics skills (e.g. quality check of reads and their alignment against the reference genome) and a certain degree of familiarity using command-line tools in a *nix environment (e.g. Linux or MacOS).

<br>
<br>

### Learning outcomes
By the end of this course, the participants will be able to:

- acquire theoretical knowledge about methods to profile RNA modifications
- use bioinformatics tools to analyze Nanopore reads
- use bioinformatics tools to pre-process RNAseq data
- use command-line tools (REDItools) for calling, annotating (REDIportal), and filtering RNA variants

<br> 
<br>

### Laptop
Participants are kindly requested to bring their own laptop. In case of need, organizers will provide participants with a computer.

<br> 
<br>

### Contact
Should you have any question, do not hesitate to contact the course Local Organizers ([elixirit.tp.bari@gmail.com](mailto:elixirit.tp.bari@gmail.com))

