---
layout: post
title: "Gene Ontology and Bioinformatic data Resources Workshop: Data Annotation and functional analysis tools."
date_start: 2023-05-03
date_end: 2023-05-04
venue: Rome, Italy
description: Two-day workshop to boost your understanding of bioinformatics resources and their use - you will learn about the key data resources available and how to use functional analysis tools.

---


<table border="0">
  <tr>
 <td><a href="http://elixir-italy.org"><img src="../../../img/logo_iib.png" height="150"></a></td>
  </tr>
</table>
<br>
<br>

## Important Dates
- Deadline for applications: **12th April 2023** 
- Chosen participants will be notified by: **17th April 2023**
- Course date: 3-4 May 2023
<br>
<br>

### Venue
University of Rome Tor Vergata

Aula 3 from 9AM to 1PM / Aula 17 from 2PM to 6PM, Department of Biology, Rome.  [**(link to the map)**](https://maps.google.com/maps?q=41.85397136127,12.605045153653) 
Via della Ricerca Scientifica 
00133 Rome, Italy
<br>
<br>

### Fee
The course does not include a fee, but the participants are expected to pay their own travel and accomodation costs (if any).
<br>
<br>

### **Selection**
A maximum of **30 candidates** will be selected on a first-come- first-served basis. Chosen participants will be notified by 17th April 2023. 

**Cancellation policy:** Attendance is limited to 30 participants. We expect many more applications. Accepted participants must commit to attend the course for its whole duration. Failure to attend training sessions is disruptive and prevents other candidates from participating. Please email the course organisers if you are unable to attend so that people on the waiting list can be invited to take your place.
<br>
<br>

### Instructors
- **Ruth Lovering** -  University College of London, UCL, UK
- **Luana Licata** - University of Rome Tor Vergata, ELIXIR-IT
<br>
<br>

### Scientific Organisers
- **Ruth Lovering** - University College of London, UCL, UK
- **Luana Licata** - University of Rome Tor Vergata, ELIXIR-IT
<br>
<br>

### Local Organisers
- **Luana Licata** - University of Rome Tor Vergata, ELIXIR-IT
<br>
<br>

### Contact
For all kinds of queries, please contact the local organiser Luana Licata at: [luana.licata@uniroma2.it](mailto:luana.licata@uniroma2.it) . For questions about course content, please email the scientific organisers [luana.licata@uniroma2.it](mailto:luana.licata@uniroma2.it) or [r.lovering@ucl.ac.uk](mailto:r.lovering@ucl.ac.uk).
<br>
<br>

### Course Description
The course will provide information about Gene Ontology (GO) and molecular interaction resources and practical demonstrations of a number of freely available high-throughput analysis and network analysis tools that can be used for the analysis of large datasets, such as those derived from proteomic or transcriptomic methodologies.
<br>
<br>

### Target audience
The workshop is open to all biological or biomedical PhD and Post-Doc research scientists, as well as team leaders and PIs. You don't need to have any previous knowledge of the tools that will be presented at the workshop. Attendees may bring in their own datasets for high-throughput analysis, although example datasets will be available.
<br>
<br>

### Aims of the workshop
•	To provide information about Gene Ontology (GO) and molecular interaction resources 
•	To provide practical demonstrations of a number of freely available bioinformatic resources and tools that can be used for the analysis of large datasets, such as those derived from proteomic or transcriptomic methodologies.
<br>
<br>

### Resources and tools covered
•	Ensembl
•	Identifier mapping
•	Gene Ontology (GO) 
•	Functional analysis tools
•	Molecular interaction resources 
•	Cytoscape and network analysis tools
<br>
<br>

### Learning Outcomes
By the end of this course, the participants will be able to:
- Identify and access public data resources and tools for data integration, high-throughput analysis, functional analysis and network analysis
- Exploit the information available in the Gene Ontology (GO) to identify key cellular processes associated with high-throughput datasets.
- Interpret the information given in different molecular interaction and signaling databases
- Compare different sources of protein-protein interaction data and causal interaction data.
- Represent networks using Cytoscape
- Identify tools used for network analysis and functional analysis
<br>
<br>  

### Course prerequisites
Participants can use the computer available in the meeting room however if they prefer, they can bring their own laptop and install before the meeting few specific software.  Materials will be provided in advance of the workshop by the organizers. 
<br>
<br>

### [Application Form - Registration](https://docs.google.com/forms/d/e/1FAIpQLSeRrEW8ScR_9iEwa4hSNu6ko45bqbHk5RhC0z67oWWe3xtNAg/viewform)
<br>
<br>
<br>

### Programme
<table border="1" width="700">
  <tr>
    <td colspan="4" align=center><h3> Wednesday - 3 May</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 09:00 - 9:20</td>
    <td height="50" align=center> Lecture</td>
    <td height="50"> What do you know about bioinformatics databases? Small group discussions and feedback to attendees</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 09:20 - 09:35</td>
    <td height="50" align=center> Lecture</td>
    <td height="50"> Brief overview of UniProt</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 09:35 - 9:45</td>
    <td height="50" align=center> Lecture</td>
    <td height="50"> Brief overview of HGNC</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 09:45 - 10:00</td>
    <td height="50" align=center> Lecture</td>
    <td height="50"> Identifier mapping</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 10:00 - 10:45</td>
    <td height="50" align=center> Lecture</td>
    <td height="50"> Gene Ontology</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 10:45 - 11:15</td>
    <td colspan="3" height="50" align=center> Coffee break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 11:15 - 12:00</td>
    <td height="50" align=center> Lecture and Exercise</td>
    <td height="50"> Browsing the GO resource with QuickGO</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 12:00 - 12:45</td>
    <td height="50" align=center> Lecture and Exercise</td>
    <td height="50"> Browsing the GO resource with AmiGO</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 12:45 - 13:45</td>
    <td colspan="3" height="50" align=center> Lunch break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 13:45 - 14:30</td>
    <td height="50" align=center> Lecture</td>
    <td height="50"> High-throughput data and functional analysis tools</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 14:30 - 15:45</td>
    <td height="50" align=center> Lecture and Exercise</td>
    <td height="50"> High-throughput data analysis using variety of tools, for example: g:Profiler,WebGestalt,DAVID,VLAD</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 15:45 - 16:15</td>
    <td colspan="3" height="50" align=center> Coffee break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 16:15 - 17:30</td>
    <td height="50" align=center> Lecture and Exercise </td>
    <td height="50"> Genome browser: Ensembl</td>
  </tr>
  <tr>
    <td colspan="4" align=center><h3>Thursday - 4 May</h3></td>
  </tr>
  <tr>
   <td height="50" width="100" align=center>09:00 - 10:30</td>
   <td height="50" align=center>Lecture</td>
   <td height="50">Protein-Protein Interactions and Directed Molecular Interactions</td>
  </tr>
  <tr>
   <td height="50" width="100" align=center>10:30 - 11:00</td>
   <td colspan="3" height="50" align=center>Coffee break</td>
  </tr>
  <tr>
   <td height="50" width="100" align=center>11:00 - 13:00</td>
   <td height="50" align=center>Lecture and Exercise</td>
   <td height="50">Interaction databases eg. IMEx, STRING, others and Signaling and pathways databases</td>
  </tr>
  <tr>
   <td height="50" width="100" align=center>13:00 - 14:00</td>
   <td colspan="3" height="50" align=center>Lunch break</td>
  </tr>
  <tr>
   <td height="50" width="100" align=center>14:00 - 15:45</td>
   <td height="50" align=center>Lecture and Exercise</td>
   <td height="50">Cytoscape, GOlorize, BinGO, others</td>
  </tr>
  <tr>
   <td height="50" width="100" align=center>15:45 - 16:15</td>
   <td colspan="3" height="50" align=center>Coffee break</td>
  </tr>
  <tr>
   <td height="50" width="100" align=center>16:15 - 17:30</td>
   <td height="50" align=center>Exercise</td>
   <td height="50">Continue Cytoscape</td>
  </tr>
</table>

<br>
<br>
<br>

### ELIXIR-IT member institutes
1. *CNR, National Research Council*(Lead Institute)
2. CRS4
3. CINECA
4. Edmund Mach Foundation, Trento
5. ENEA
6. Fondazione Telethon
7. INFN
8. Istituto Superiore di Sanità (ISS)
9. GARR
10. Stazione Zoologica Anton Dohrn, Napoli
11. University of Roma "Sapienza"
12. University of Bari
13. University of Bologna
14. University of Firenz
15. University of Milano
16. University of Milano Bicocca
17. University of Napoli
18. University of Padova
19. University of Parma
20. University of Roma "Tor Vergata"
21. University of Salern
22. University of Torino
23. University of Tuscia
