---
layout: default
---

<div class="home">

  <h3 class="page-heading">Programme 2016</h3>
  <ol>
    <li><b>High Performance Molecular Dynamics</b><br>
      <i>CINECA, via dei Tizii 6, Rome - 5-7 April, 2016</i><br>
This course is designed for those users who wish run classical molecular dynamics programs such as GROMACS and NAMD on modern supercomputers. By understanding better the HPC infrastructures and the algorithms used to exploit them, the aim is to give researchers the tools to run simulations in the most efficient way possible on current and future supercomputers. 
At least half the course will be devoted to practical sessions where students will be able to prepare and run GROMACS or NAMD examples on the supercomputers of Cineca.
</li>
<br>
<li><b>NGS for evolutionary biologists: from basic scripting to variant calling</b><br>
<i>Biblioteca, Consiglio Nazionale delle Ricerche, via P. Castellino 111, 80131 Naples - May 2-6, 2016</i><br>
This course will provide an introduction to next generation sequencing platforms, data analysis and tools for data quality control, including alignment to a reference sequence, data handling and visualisation, and variant calling and filtering (single nucleotide polymorphisms and structural variants). The course will be delivered using a mixture of lectures and computer based hands-on practical sessions, including group projects to be completed by the participants using the knowledge gained at the course. Projects will cover the general topics of population structure and admixture, demographic changes and natural selection.
</li>
<br>
<li><b>RNA-seq data analysis workshop</b><br>
<i>University of Naples Federico II, Naples - June 6-10, 2016</i><br>
This course will provide an introduction on the application of the RNA-seq technology to the study of non-model organism transcriptomes and it will be centered on the Trinity software suite for the de novo transcriptome assembly and analysis. The course will be delivered by the lead developer of the Trinity software Dr. Brian Haas (the Broad Institute, USA) and by tutors from the University of Naples Federico II, the Institute of Genetics and Biophysics "Adriano Buzzati-Traverso", the Sapienza University of Rome and the National Research Council (CNR). It will consist of theoretical lessons and practical exercises on transcriptome assembly, structural and functional annotation and differential expression analysis.
</li>
<br>
<li><b>Effective academic teaching</b><br>
<i>University of Salerno, Fisciano (SA), Italy - June 15th, 2016</i><br>
In this workshop, we will provide participants with instructional principles that come directly from cognitive research and the speakers' experience, and their implications for teaching practice. We will then extensively discuss which principles can be easily translated (and how) into teaching strategies and applied into actual courses. Examples of effective teaching techniques already put into action in academic courses will be described. 
</li>
<br>
<li><b>Exome analysis using Galaxy</b><br>
<i>Università di Milano-Bicocca, Laboratory 4A1, Building U4, Piazza della Scienza 4, 20126, Milano (IT) - September 19-20, 2016</i><br>
Galaxy is an open source, web-based platform for data intensive biomedical research. It makes accessible bioinformatics applications to users lacking programming skills, enabling them to easily build analysis workflows for NGS data.
This course is aimed at PhD student, biologists, clinicians and researchers who are analysing, or need to analyse in the near future, high throughput exome sequencing data.
The aim of this course is to make participants familiarise with the Galaxy platform and prepare them to work independently, using state-of-the art tools for the analysis of exome sequencing data.
</li>
<br>
<li><b>High Performance Molecular Dynamics@CINECA - PRACE Edition</b><br>
<i>CINECA, Bologna - 20-30 September, 2016</i><br>
This course is designed for those users who wish run classical molecular dynamics programs such as GROMACS and NAMD on modern supercomputers. By understanding better the HPC infrastructures and the algorithms used to exploit them, the aim is to give researchers the tools to run simulations in the most efficient way possible on current and future supercomputers. 
At least half the course will be devoted to practical sessions where students will be able to prepare and run GROMACS or NAMD examples on the supercomputers of Cineca.
</li>
<br>
<li><b>Python for Life Scientists</b><br>
<i>Aula Multimediale, Labo-Biotech - Department of Biosciences, Biotechnology and Biopharmaceutics, University of Bari - Via Fanelli 204 - September 26-30, 2016</i><br>
Python is an object-oriented programming language that is ideal for biological data analysis. The course will start from zero knowledge, and will introduce the participants to all the basic concepts of Python such as calculating, organising data, reading and writing files, program logic and writing larger programs. All the examples and practical sessions will focus on solving biological problems. 
The course will be highly interactive and the students will continuously put theory into practice while learning. By the end of the course, the participants will have a good understanding of Python basics and will have acquired the skills to manage any type of bioinformatics database record and to run applications from scripts. Basic Unix/Linux skills will be provided at the beginning of the course.
</li>
<br>
<li><b>GOBLET/ELIXIR-IIB Train-The-Trainer Tutorial (ELIXIR/EXCELERATE TtT 3rd pilot course)</b><br>
<i>Consiglio Nazionale delle RIcerche (CNR) - Piazzale Aldo Moro, 7 - Rome, IT - 24th October, 2016</i><br>
In this workshop, we will provide participants with instructional principles that come directly from cognitive research and the speakers' experience, and their implications for teaching practice. The workshop, which is limited to 20 participants, is organised in four sessions: Session 1: Review of learning principles and how they apply to training; Session 2: Training techniques that can be used to enhance learner engagement and participation; Session 3: Session, course, and materials design; Session 4: Assessment and feedback in training.
</li>
<br>
<li><b>Combined CHARME - EMBnet - NETTAB 2016 Workshop</b><br>
<i>Consiglio Nazionale delle RIcerche (CNR) - Piazzale Aldo Moro, 7 - Rome, IT - 25-26 October 2016</i><br>
The Workshop ‘Reproducibility, standards and SOP in bioinformatics‘ is co-organised by the COST European Action CHARME (CA15110), EMBnet (The Global Bioinformatics Network) and NETTAB (International Workshop Series on Network Tools and Applications for Biology). It is hosted by the ELIXIR Italian Node (ELIXIR-IIB)) and will be held at the Italian CNR (National Research Council) head quarter, in Rome. The workshop will be preceded by a GOBLET/ELIXIR-ITA Tutorial and a ELIXIR Hacktahon on Monday 24th.
</li>
</ol>
</div>
